pkg/asset/manifests/ibmcloud/cloudproviderconfig.go

```diff
@@ -40,6 +44,9 @@ func CloudProviderConfig(infraID string, accountID string) (string, error) {
 			AccountID:                accountID,
 			ClusterID:                infraID,
 			ClusterDefaultProvider:   "g2",
+			Region:                   region,
+			G2CredentialsFilePath:    "/etc/vpc/ibmcloud_api_key",
+			G2VPCName:                fmt.Sprintf("%s-vpc", infraID),
```

Comment:
wanted to double, check, but this looks as expected
https://github.com/openshift/installer/blob/431adc5a693de09eac93f96492c7a51f792fb223/data/data/ibmcloud/network/vpc/vpc.tf#L1-L14